pkg/providers/storage.go

```diff
@@ -5,14 +5,36 @@ import (
 	"github.com/weaveworks/ignite/pkg/constants"
 	"github.com/weaveworks/ignite/pkg/storage"
 	"github.com/weaveworks/ignite/pkg/storage/cache"
+	"github.com/weaveworks/ignite/pkg/storage/manifest"
+	"github.com/weaveworks/ignite/pkg/storage/sync"
+	"github.com/weaveworks/ignite/pkg/storage/watch"
 )
 
 // Storage is the default storage implementation
 var Storage storage.Storage
+var SS *sync.SyncStorage
```

Comment:
rename this to something better, SS doesn't say me anything

Reply:
It's `SyncStorage` now, added comments to it as well